src/sparse_variational.jl

```diff
@@ -1,16 +1,41 @@
+# Parametrisations.
+struct Centred end
+struct NonCentred end
+
+struct SparseVariationalApproximation{Parametrisation,Tfz<:FiniteGP,Tq<:AbstractMvNormal}
+    fz::Tfz
+    q::Tq
+end
+
 """
-    SparseVariationalApproximation(fz::FiniteGP, q::AbstractMvNormal)
+    SparseVariationalApproximation(::Parametrisation, fz::FiniteGP, q::AbstractMvNormal)
```

Comment:
Maybe we need some additional docstring to explain what are the Centred/NonCentred parametrizations?

Reply:
Done. Please let me know if it's unclear.

Reply:
I would maybe just add the equivalence with "whitened"/"not whitened" that people might be more used to

Reply:
Added.

Reply:
Thanks! Looks all good to me now. Do we actually want to also have aliases with `const Whitened = NonCentred` etc?

Reply:
I'd rather not -- its just an additional thing for people to get confused by. I'd rather have a single name, and I think centred vs non-centred is probably the way to go (to my mind it's more obvious where the term comes from than whitened vs un-whitened).

Reply:
My only gripe with Centred vs Non-Centred is that I can never remember which is which :D But maybe that will be a good incentive to get better at this :D

Reply:
> My only gripe with Centred vs Non-Centred is that I can never remember which is which :D But maybe that will be a good incentive to get better at this :D

The trick is to remember it's the opposite, centred = non-whitened, non-centred = whitened :joy: 

> (to my mind it's more obvious where the term comes from than whitened vs un-whitened).

I have to admit that's still not quite obvious to mine! care to explain? : )

Reply:
At least in the Gaussian case, the distribution is "centred on the mean parameter" in the sense that its mode is at the mean, whereas in the non-centred case it's "not centred on the mean parameter", in the sense that the mode of the RV of interest is at `m_prior + sqrt(cov_prior) * m_approx_post`.

At least that's how I interpret it. I agree that for arbitrary distributions it's less obvious.